pantheon/src/test/java/tech/pegasys/pantheon/cli/operator/OperatorSubCommandTest.java

```diff
@@ -152,15 +153,36 @@ public void privateKeyFileNameShouldBeEqualToOption() throws IOException {
         asList("key.pub", "priv.test"));
   }
 
-  @Test(expected = CommandLine.ExecutionException.class)
-  public void shouldFailIfDuplicateFiles() throws IOException {
-    runCmdAndCheckOutput(
-        cmd("--private-key-file-name", "dup.test", "--public-key-file-name", "dup.test"),
-        "/operator/config_generate_keys.json",
-        tmpOutputDirectoryPath,
-        "genesis.json",
-        true,
-        asList("key.pub", "priv.test"));
+  @Test
```

Comment:
Did this sneak in from another clangelist?

Reply:
No, I made the validation stricter [here](https://github.com/PegaSysEng/pantheon/pull/1809/files#diff-0463d13bbf95f450907e60b2d512ddf8R187), so added a test for that, and reworked this pre-existing test to use `assertThatThrownBy` rather than the `@Test(expected = ...)` annotation.

This subcommand was using `vertx` libraries because it was interacting with `JsonGenesisConfigOptions`.